.github/workflows/update-dependencies.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           repository: nanoframework/nf-tools
           path: tools
       - name: Update dependencies
-        run: ./github-actions/update-nf-dependencies.ps1 prerelease devices
+        run: ./github-actions/update-nf-dependencies.ps1
```

Comment:
OK for using only stable versions on this, but it still has to be run on "devices" folder only. Otherwise it will end trying to update projects outside the devices folder.

Reply:
Should be fixed now, by defining the `targetPath` variable.